resources/problem-vote.scss

```diff
@@ -0,0 +1,54 @@
+.vote-form {
```

Comment:
Annoying nit: can we prefix all of these with `problem-vote-` rather than just `vote-`, to disambiguate from comment votes?